frontend/src/app/pages/home/home.component.html

```diff
@@ -7,3 +7,5 @@
     class="item"
     (click)="triggerDetails(item)">{{ item }}</li>
 </ul>
+
```

Comment:
If you've merged master, you missed that there should have been a `<grade-level-selection></grade-level-selection>` after the header. Your tags should have been after that.